app.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import route from './routes/routes';
```

Comment:
As for me `./routes/routes` a little bit confusing. Let's use the power of `require`! :)
```suggestion
// before that we should rename `routes.js` file to `index.js`
import route from './routes';
```